libs/react-components/package.json

```diff
@@ -5,9 +5,13 @@
   "main": "index.js",
   "module": "index.js",
   "exports": {
+    "./tedi": {
+      "types": "./tedi/index.d.ts",
+      "import": "./index.js"
+    },
     "./community": {
       "types": "./community/index.d.ts",
-      "import": "./index.js"
+      "import": "./index2.js"
```

Comment:
index2 name is not very good approach. Can it be just index.js? Or otherwise pehaps community.js and tedi.js respectively.